### ISSUE TITLE
How to get openset accuracy?

### ISSUE THREAD

**Issue by Qoo95** [zhmiao/OpenLongTailRecognition-OLTR#48]
Thanks for sharing your work. May I ask how to calculate openset accuracy? Because, before calculating openset accuracy, it has 
```python
probs, preds = F.softmax(self.total_logits.detach(), dim=1).max(dim=1)
```
so that open_threshold doesn't work.
https://github.com/zhmiao/OpenLongTailRecognition-OLTR/blob/fb6203c8e1cfa7ed5bf9b1acf581800d04f92c4a/run_networks.py#L296

**Comment by zhmiao** [comment]
Hello @Qoo95 , the openset accuracy is calculated here: https://github.com/zhmiao/OpenLongTailRecognition-OLTR/blob/master/run_networks.py#L297

Open threashold won't be used if main argument `test_open` is not True. 

Does that make sense?

**Comment by zhmiao** [comment]
It has been 26 days since last comment. I will close this issue now. If you have any more questions, please reopen this issue. Thanks.

**Comment by twmht** [comment]
@zhmiao 

so if `probs < self.training_opt['open_threshold']` is equal to true, this input image is regard as an open set input (novel class)?
